tests/e2e/e2e-test-in-kind.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -x
+
+# Copyright 2019 The xridge kubestone contributors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This is a modified version of kind deploy script, taken from:
+# https://github.com/kind-ci/examples
+
+set -eEuo pipefail
+
+BIN_DIR="$(mktemp -d)"
+KIND="${BIN_DIR}/kind"
+KUBECTL="${BIN_DIR}/kubectl"
+
+cleanup() {
+    kubectl get all --all-namespaces || true
+    kubectl logs -n kubestone-system $(kubectl get pods -n kubestone-system -o name | grep controller) || true
```

Comment:
I suppose this is due to lack of labels on the deployment. 
If so we should add a TODO and capture that deficiency.

Reply:
spot on. added a todo to not to forget.